daemon/daemon.go

```diff
@@ -719,8 +710,10 @@ func NewDaemon(config *Config, registryService *registry.Service, containerdRemo
 		return nil, err
 	}
 
-	d.downloadManager = xfer.NewLayerDownloadManager(d.layerStore, maxDownloadConcurrency)
-	d.uploadManager = xfer.NewLayerUploadManager(maxUploadConcurrency)
+	logrus.Debugf("Max Concurrent Downloads: %d", *config.MaxDownloads)
```

Comment:
I notice that the pointers aren't checked for nil in this function. I *think* that's okay, because when this function is called, they should already have been initialized by `InstallCommonFlags`. Is that right?